eventsource/stores/memorystore/store.go

```diff
@@ -32,6 +32,26 @@ func (mem *store) LoadBySequenceID(_ context.Context, sequenceID string) (record
 			if row.SequenceID > sequenceID {
 				records = append(records, row)
 			}
+			if len(records) >= 1000 {
```

Comment:
This will make this function pretty much "useless" without pagination, why is this needed?

Reply:
The alarm status event table has > 200000 records. Unmarshalling them all is time consuming, and results in messages that are far too big to send over gRPC. Also, doing something meaningful with 200000 records in a lambda takes too much time. I realize that these limitations are mostly related to my specific use case - maybe the limit should be given as a parameter?